packages/mobile/locales/en-US/nuxNamePin1.json

```diff
@@ -26,6 +29,7 @@
   "fullName": "Full Name",
   "importIt": "Restore Existing Wallet",
   "cancel": "Cancel",
+  "accept": "Accept",
```

Comment:
I think Spanish translation for this is missing

Reply:
how do you typically handle getting the spanish? online robo translator?

Reply:
For simple stuff - just G Translator, for more complicated phrases - channel in Slack #spanish